src/code/audio/code_801A5BD0.c

```diff
@@ -14,7 +14,7 @@ typedef struct {
     /* 0x4 */ f32 target;
     /* 0x8 */ f32 step;
     /* 0xC */ u16 remainingFrames;
-} UnusedBankLerp; // size = 0x10
+} SfxBankLerp; // size = 0x10
 
 extern u8 D_801D6600[7];
 extern u8 D_801D6608[7];
```

Comment:
Is it known what these are?

Reply:
It's true-false data for each of the 7 sound-banks:
https://github.com/zeldaret/mm/blob/ad42edb0340ba5e0e2f6e318b58fcb0af61312d1/src/code/audio/code_8019AF00.c#L156-L162

In OoT, there's only 1, not 2, and it's called `gIsLargeSoundBank`, but I don't think that name is accurate to begin with, so I refrained naming these.
https://github.com/zeldaret/oot/blob/32e66c2da8c2f9816773e1b2a0bc92804e9d20de/src/code/code_800EC960.c#L87

Nonetheless, these externs should be in variables.h I just noticed